src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,27 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Comparator<Employee> employeeComparator = new Comparator<Employee>() {
+            @Override
+            public int compare(Employee o1, Employee o2) {
+                int ageCompared = Integer.compare(o1.getAge(), o2.getAge());
+
+                if (ageCompared != 0) {
+                    return ageCompared;
+                }
+
+                return o1.getName().compareTo(o2.getName());
+            }
+        };
+
+        Set<Employee> sortedEmployees = new TreeSet<>(employeeComparator);
```

Comment:
Directly using `TreeSet` with a custom comparator is fine, but you may also consider using `Stream` API from the `List` interface for sorting purposes. This can sometimes result in more readable code, especially for more complex sorting criteria.